install/gains.csv

```diff
@@ -17,6 +17,9 @@ CPXB,27,,,0.1,,,2010-05-01T00:00:00Z,2015-05-07T02:00:00Z
 CPXB,28,,,0.1,,,2010-05-01T00:00:00Z,2015-05-07T02:00:00Z
 CPXB,29,,,0.1,,,2010-05-01T00:00:00Z,2015-05-07T02:00:00Z
 CPXB,2A,,,0.1,,,2010-05-01T00:00:00Z,2015-05-07T02:00:00Z
+EY2M,50,,X,,0x67,,2024-04-042T16:00:00Z,9999-01-01T00:00:00Z
+EY2M,50,,Y,,0x00,,2024-04-042T16:00:00Z,9999-01-01T00:00:00Z
+EY2M,50,,Z,,0x8c,,2024-04-042T16:00:00Z,9999-01-01T00:00:00Z
```

Comment:
to review these, can you please provide a photo of the instrument setup in the equipment change ticket?

Reply:
Amended and added photo of gains to ticket https://github.com/GeoNet/tickets/issues/15482